array.c

```diff
@@ -5726,14 +5749,12 @@ rb_ary_hash(VALUE ary)
 
 /*
  *  call-seq:
- *     ary.include?(object)   -> true or false
+ *    array.include?(obj) -> true or false
  *
- *  Returns +true+ if the given +object+ is present in +self+ (that is, if any
- *  element <code>==</code> +object+), otherwise returns +false+.
- *
- *     a = [ "a", "b", "c" ]
- *     a.include?("b")   #=> true
- *     a.include?("z")   #=> false
+ *  Returns +true+ if for some index +i+ in +self+, <tt>obj == self[i]</tt>;
```

Comment:
why does it matters the internals of the index with the fact given in the previous paragraph and examples. Is it needed?

Reply:
I don't understand.  This is what I see:

/*
 *  call-seq:
 *    array.include?(obj) -> true or false
 *
 *  Returns +true+ if for some index +i+ in +self+, <tt>obj == self[i]</tt>;
 *  otherwise +false+:
 *    [0, 1, 2].include?(2) # => true
 *    [0, 1, 2].include?(3) # => false
 */

No previous paragraph.

Reply:
I meant what it says before:
```
*  Returns +true+ if the given +object+ is present in +self+ (that is, if any
*  element <code>==</code> +object+), otherwise returns +false+.
```
There's no mention of an index, just said "if .. is present". Maybe something like 
- "is present in the items"
- "is part of the collection of +self+"

but not sure what would be better phrasing

I'm now also seeing that's how is mention in other areas, so I think it should be fine.

Reply:
ok